source-gallery/src/main/java/com/google/android/apps/muzei/gallery/GallerySettingsActivity.java

```diff
@@ -444,7 +429,7 @@ public void run() {
         return super.onOptionsItemSelected(item);
     }
 
-    private void requestGetContent(ActivityInfo info) {
```

Comment:
Add @Override annotation

Reply:
What do you mean? I am sure I added the Override annotation

Reply:
I added the annotation. Please see line 408

Reply:
Got it, I see it now.

Reply:
No problem.